### PR TITLE
correct documented 'git clone' behaviour

### PR DESCRIPTION
**Pull request by jim-minter**
xref https://bugzilla.redhat.com/show_bug.cgi?id=1533129

### CODE REVIEW

**Comment by jim-minter** [comment]
@bmcelvee @bparees @mfojtik fyi

**Comment by bparees** [comment]
lgtm

File: dev_guide/builds/build_inputs.adoc [hunk 1]
```diff
@@ -216,22 +216,15 @@ containing a Dockerfile that overwrites any Dockerfile that may exist in the
...
 and then checkout `FETCH_HEAD`.
 
-To perform a full `git clone` of the `master` for the specified repository, set
-the `ref` to `master`.
+When no `ref` value is provided, {product-title} performs a shallow clone
+(`--depth=1`).  In this case, only the files associated with the most recent
+commit on the default branch (typically `master`) are downloaded.  This results
+in repositories downloading faster, but without the full commit history.  To
+perform a full `git clone` of the default branch of a specified repository, set
+`ref` to the name of the default branch (e.g. `master`).
```
Comment (bmcelvee) [inline dev_guide/builds/build_inputs.adoc]: s/e.g./for example,

Unfortunately our style guide doesn't allow for Latin abbreviations. 😿

**Comment by bmcelvee** [comment]
Just one tiny nit. Otherwise, LGTM! Is this going into 3.5 forward?

**Comment by jim-minter** [comment]
Updated - although I note that our docs do already contain a great deal of i.e., e.g. and etc., not to mention a few vs...

**Comment by jim-minter** [comment]
3.5+ would be great.

**Comment by bparees** [comment]
@bmcelvee bump

**Comment by bmcelvee** [comment]
[rev_history]
|xref:../dev_guide/builds/build_inputs.adoc#dev-guide-build-inputs[Build Inputs]
|Updated `git clone` behavior in the xref:../dev_guide/builds/build_inputs.adoc#source-code[Git Source] section.
%

**Pull request merged**